### ISSUE TITLE
Apple Home: All devices being recreated when home hub changes

### ISSUE THREAD

**Issue by TheTeeDot** [Luligu/matterbridge#190]
Hi,
Since iOS18.2 I have noticed that when the home hub changes all of the devices I have configured are removed and new devices are created.  This results in the loss of any configuration of the device (Assignment to room etc) as well as removal from any automations.

I am only running the Shelly plugin and have 11 Shelly devices.

Is there any guidance on troubleshooting?  I am not seeing any errors in the logs but I do not have this set to debug

TIA

**Comment by Luligu** [comment]
Hi,
this can happen only if have a configuration issue.
I should see your logs and configuration to help you understand what is wrong in your setup.

Also need to know where and how you run Matterbridge.

**Comment by TheTeeDot** [comment]
Sorry, I should have included some of that detail

Matterbridge running on Raspian (RPi4).  Matterbridge 1.6.7 with the 1.1.0 Shelly plugin.  MatterBridge is set up to start via systemctl with only one configuration change to the Shelly plugin for the contact sensor to be exposed.  Let me know if you want to know any other specific configuration

I noted another thread to remove the resumption records, which I tried and had no success.

In the interim I have performed a factoryreset and will see if the problem persists and get some more detailed logs.  I dont want to specify the home hub if I can help it but that may be the answer.  I also note that specifying an IP address for the Shellys might make an impact but I am confident my DHCP server is providing the same IP to the devices but that is a step I can also take to rule out that as a factor
